public/admin/config.yml

```
@@ -243,3 +246,37 @@ collections:
                   name: "url",
                   widget: "string"
                 }
+  - name: "donasi_page"
+    label: "Donasi Page"
+    delete: false
+    preview_path: "donasi"
+    files:
+      - name: "informasi_donasi"
+        label: "Informasi Donasi"
+        file: "_content/donasi/informasi-donasi.json"
```

```suggestion
        file: "_content/donasi.json"
```

Please simplify the file structure, thanks!